data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,391 @@ mission "Remnant: Face to Maw 2B"
+				accept
+	on visit
+		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
+	on complete
+		payment 150000
+		conversation
+			`When you and the <npc> settle back onto the landing pad there is already a dockworker waiting with a Key Stone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the Keystone to a lift and disconnects it from the ship. As the Keystone is hauled up to the roof another lift lowers the replacement Key Stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`
+
+
+
+mission "Remnant: Crystal Research 6"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the Key Stone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
+		system "Peragenor"
+		ship "Starling" "Winds of Light"
+	to offer
+		has "Remnant: Crystal Research 5: done"
+	on offer
+		conversation
+			`	"Ready? Let's go!" signs Aeolus as he fires up his systems and gestures an acknowledgement to the dockworker. The worker nimbly steps clear of the ship and watches as the two ships head back out of the dome.`
+				launch
+	on visit
+		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
+	on complete
+		payment 150000
+		conversation
+			`The scanning proceeds smoothly, and while it does note some differences between the two crystals, they both appear to function similarly. After you return to Viminal, Aeolus comes over to collect a copy of the data for the researchers.`
+			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully "Perhaps more importantly, it may help understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aids."`
+			`	He looks down at the data crystal. "I have one more task to ask of you. Meet me in the cafeteria in an hour to discuss it. May the Embers burn bright for you, Captain." He gives a wave that seems vaguely like a salute and heads off.`
+			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so they must have been combining them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`
```

Comment:
`been combining` would sound better as `combined` in my opinion.
Sidenote: I love it whenever someone uses "however" midsentence instead of the beginning. Don't know if you read Strunk and White in HS/college, but that usage was like the one thing I took away from that book.

Reply:
Changed.

Thanks. I have a tendency to overuse "however", but I am glad that I used it in a way that is appreciated in this case.